misago/richtext/plugins/pretty_text.py

```diff
@@ -0,0 +1,30 @@
+__all__ = [
+    "color",
+    "size",
+]
+
+
+COLOR = r'\[color="((\w)*)"\](.*)\[\/color\]'
+SIZE = r"\[size=((\d)*)\](.*)\[\/size\]"
```

Comment:
Lets keep this here, but eventually this should be extracted to top-level `plugins` directory and be opt-in feature.